<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DrawingBoard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/new/color/src/Main.png</normaloff>:/new/color/src/Main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,10" columnstretch="1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1">
      <item row="0" column="0">
       <widget class="QPushButton" name="Point">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawPoint.png</normaloff>:/new/color/src/drawPoint.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawLine.png</normaloff>:/new/color/src/drawLine.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Curve">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawCurve.png</normaloff>:/new/color/src/drawCurve.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="Rect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawRect.png</normaloff>:/new/color/src/drawRect.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="FilledPoligon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>T</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="Poligon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawPoligon.png</normaloff>:/new/color/src/drawPoligon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="Ellipse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawEllipse.png</normaloff>:/new/color/src/drawEllipse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="Circle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/drawCircle.png</normaloff>:/new/color/src/drawCircle.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="ChooseByPid">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/choose.png</normaloff>:/new/color/src/choose.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="ChooseRect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/ChooseRect.png</normaloff>:/new/color/src/ChooseRect.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="ChooseInvert">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/ChooseInvert.png</normaloff>:/new/color/src/ChooseInvert.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QPushButton" name="Move">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/Move.png</normaloff>:/new/color/src/Move.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QPushButton" name="Rotate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/Rotate.png</normaloff>:/new/color/src/Rotate.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QPushButton" name="Zoom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/Zoom.png</normaloff>:/new/color/src/Zoom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QPushButton" name="Delete">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/deleteZone.png</normaloff>:/new/color/src/deleteZone.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QPushButton" name="Undo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/undo.png</normaloff>:/new/color/src/undo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="16">
       <widget class="QPushButton" name="Redo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/redo.png</normaloff>:/new/color/src/redo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="0" column="17">
       <widget class="QComboBox" name="PointSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>1</string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="18">
       <widget class="QPushButton" name="ButtonChooseColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/new/color/src/Color.png</normaloff>:/new/color/src/Color.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="19">
       <widget class="OpenGLWindow" name="openGLWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="newPaint"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>插入</string>
    </property>
    <addaction name="drawPoint"/>
    <addaction name="drawLine"/>
    <addaction name="drawCurve"/>
    <addaction name="drawCircle"/>
    <addaction name="drawEllipse"/>
    <addaction name="drawRect"/>
    <addaction name="drawPoligon"/>
   </widget>
   <widget class="QMenu" name="menuChoose">
    <property name="title">
     <string>选择</string>
    </property>
    <addaction name="chooseByPid"/>
    <addaction name="chooseRect"/>
    <addaction name="choosePologon"/>
    <addaction name="separator"/>
    <addaction name="chooseInvert"/>
    <addaction name="separator"/>
    <addaction name="move"/>
    <addaction name="rotate"/>
    <addaction name="zoom"/>
    <addaction name="deleteZone"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="undo"/>
    <addaction name="redo"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>显示</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInsert"/>
   <addaction name="menuChoose"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="drawLine">
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="drawPoint">
   <property name="text">
    <string>画点</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="drawCurve">
   <property name="text">
    <string>曲线</string>
   </property>
  </action>
  <action name="drawCircle">
   <property name="text">
    <string>圆形</string>
   </property>
  </action>
  <action name="drawEllipse">
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="drawRect">
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="drawFilledRect">
   <property name="text">
    <string>填充矩形</string>
   </property>
  </action>
  <action name="drawPoligon">
   <property name="text">
    <string>多边形</string>
   </property>
  </action>
  <action name="drawFilledPoligon">
   <property name="text">
    <string>填充多边形</string>
   </property>
  </action>
  <action name="chooseRect">
   <property name="text">
    <string>矩形区域</string>
   </property>
  </action>
  <action name="choosePologon">
   <property name="text">
    <string>多边形区域</string>
   </property>
  </action>
  <action name="chooseInvert">
   <property name="text">
    <string>反选</string>
   </property>
  </action>
  <action name="move">
   <property name="text">
    <string>移动</string>
   </property>
  </action>
  <action name="rotate">
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="zoom">
   <property name="text">
    <string>缩放</string>
   </property>
  </action>
  <action name="undo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="redo">
   <property name="text">
    <string>恢复</string>
   </property>
  </action>
  <action name="newPaint">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionViewCode">
   <property name="text">
    <string>查看源代码</string>
   </property>
  </action>
  <action name="deleteZone">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="chooseByPid">
   <property name="text">
    <string>选择</string>
   </property>
  </action>
  <action name="action_3D_plus">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_3D_minus">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenGLWindow</class>
   <extends>QOpenGLWidget</extends>
   <header>OpenGLWindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
